Sources/SwiftParser/Attributes.swift

```diff
@@ -67,82 +67,61 @@ extension Parser {
     case transpose
 
     init?(lexeme: Lexer.Lexeme, experimentalFeatures: Parser.ExperimentalFeatures) {
-      switch PrepareForKeywordMatch(lexeme) {
-      case TokenSpec(._alignment): self = ._alignment
-      case TokenSpec(._backDeploy): self = ._backDeploy
-      case TokenSpec(._cdecl): self = ._cdecl
-      case TokenSpec(._documentation): self = ._documentation
-      case TokenSpec(._dynamicReplacement): self = ._dynamicReplacement
-      case TokenSpec(._effects): self = ._effects
-      case TokenSpec(._expose): self = ._expose
-      case TokenSpec(._implements): self = ._implements
-      case TokenSpec(._nonSendable): self = ._nonSendable
-      case TokenSpec(._objcImplementation): self = ._objcImplementation
-      case TokenSpec(._objcRuntimeName): self = ._objcRuntimeName
-      case TokenSpec(._optimize): self = ._optimize
-      case TokenSpec(._originallyDefinedIn): self = ._originallyDefinedIn
-      case TokenSpec(._private): self = ._private
-      case TokenSpec(._projectedValueProperty): self = ._projectedValueProperty
-      case TokenSpec(._semantics): self = ._semantics
-      case TokenSpec(._specialize): self = ._specialize
-      case TokenSpec(._spi): self = ._spi
-      case TokenSpec(._spi_available): self = ._spi_available
-      case TokenSpec(._swift_native_objc_runtime_base): self = ._swift_native_objc_runtime_base
-      case TokenSpec(._typeEraser): self = ._typeEraser
-      case TokenSpec(._unavailableFromAsync): self = ._unavailableFromAsync
-      case TokenSpec(.`rethrows`): self = .rethrows
-      case TokenSpec(.attached): self = .attached
-      case TokenSpec(.available): self = .available
-      case TokenSpec(.backDeployed): self = .backDeployed
-      case TokenSpec(.derivative): self = .derivative
-      case TokenSpec(.differentiable): self = .differentiable
-      case TokenSpec(.exclusivity): self = .exclusivity
-      case TokenSpec(.freestanding): self = .freestanding
-      case TokenSpec(.inline): self = .inline
-      case TokenSpec(.objc): self = .objc
-      case TokenSpec(.Sendable): self = .Sendable
-      case TokenSpec(.transpose): self = .transpose
+      switch lexeme.rawTokenKind {
+      case .keyword:
+        switch Keyword(lexeme.tokenText) {
+        case .rethrows: self = .rethrows
+        default:
+          return nil
+        }
+      case .identifier:
+        switch lexeme.tokenText {
+        case "_alignment": self = ._alignment
```

Comment:
I'm leaning toward making `TokenSpec` for fixed text. I.e. `TokenSpec("_alignment")`.